stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/channels/base/Buffer.java

```diff
@@ -72,33 +72,17 @@ public Buffer(int minimumSizeOfBuffer) {
      * @param totalNumberOfBytesRequired number of bytes required.
      * @return new ByteBuffer which will contain bytes which are remaining.
      */
-    private ByteBuffer getBytesLeftInBuffer(int totalNumberOfBytesRequired) {
-        ByteBuffer bufferedContent = null;
+    private ByteBuffer remainingContent(int totalNumberOfBytesRequired) {
```

Comment:
Shall we rename this to getRemainingContent?

Reply:
Since the API signature is clear on it's return. Explicitly using "get" might not make sense at this point ?